### DISCUSSION TITLE
Extract texture intensity from inverse pole figures (IPF)

### DISCUSSION THREAD

**Discussion by matthiaskrug7** [mtex-toolbox/mtex#859] [Ask Anything]
![Bas6_mgc9_pSt-PF-IPF](https://user-images.githubusercontent.com/52535967/111492420-f0a40280-873c-11eb-9cbb-407bb1a436cb.png)

Hello,

I tried to check the intensities of certain orientations from inverse pole figures by clicking on the respective position in the IPF. In the example above (left figure), I clicked on the [-111] position, which gives me the intensity 2.3. Note that the projection is along X. However, when I do the same thing in the regular pole figure (right figure), I get a different intensity value although it is the same position. Additionally, the "calcHKL" command says that [-111] is at a totally different location (blue point, left figure). Both PF and IPF plots were generated using the default plotting options (equal area) but the same mismatch occurs when I plot them in equal angle projection. I use MTEX version 5.4.

I suspect that the positioning in the IPF (when I click on it) doesn´t work correctly so the intensity is correct but not the position. Does anybody know?

Thanks in advance!
Matthias

**Comment by kilir** [comment]
Hi,
 the cursor reports [-111] while you annotate (-111). Depending on your crystalsymmetry, those are not the same and hence end up at different positions.
Cheers,
Rüdiger

**Reply by matthiaskrug7** [reply]
Hi Rüdiger,

That explains it. Thank you very much for your reply!

Matthias

**Comment by makeabhishek** [comment]
Hi Matthias,
I have Euler angles for different x and y position obtained from EBSD for Titanium 64 alloy. I want to plot smoothened pole figure (as shown in your image) and inverse pole figure from EBSD data. Could you please share the script to plot it?
I also want to see the MUD value for the pole figure. Is it possible?
How to select crystal symmetry for Ti-64 alloy which has dual phases alpha and beta with hexagonal and cubic symmetr.
Regards
Abhishek

**Reply by matthiaskrug7** [reply]
Hi Abhishek,

I am not working with EBSD data. But there is plenty of documentation about it in the wiki. That should help you much more than I can:
https://mtex-toolbox.github.io/EBSD2ODF.html

The (simplest) code for plotting an inverse pole figure with a color scale is this: 
```
figure
  plotIPDF(odf,[xvector],'antipodal')
  mtexColorbar
```
There are also lots of more options described in the wiki, such as different colors, styles, etc.

Also try out the Import Wizard. It does 90% of the work for you, including the determination of the crystal system.

Best of luck!
Matthias
